services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)

```

Comment:
The `get_movie_by_id` function uses `Movie.objects.get(id=movie_id)`, which will raise a `DoesNotExist` exception if no movie with the given ID exists. Consider using `get_object_or_404` from `django.shortcuts` if this function is used in a view, or handle the exception appropriately to avoid potential crashes.